modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/index/hash/io/HashIndexTreeIo.java

```diff
@@ -225,28 +238,22 @@ default int compare(DataPageReader dataPageReader, int partitionId, long pageAdd
      */
     default HashIndexRow getRow(DataPageReader dataPageReader, int partitionId, long pageAddr, int idx)
             throws IgniteInternalCheckedException {
-        int off = offset(idx);
+        final int off = offset(idx);
```

Comment:
Why making it explicitly final?

Reply:
So that there is less chance of shooting yourself in the foot, as I did when reworking.